ui/src/external/monaco.fluxLang.ts

```diff
@@ -1,7 +1,9 @@
+import {FLUXLANGID} from 'src/types'
```

Comment:
nit: `FLUXLANGID` isn't a type its a value.  Should probably live in a `constants` folder.